constants.py

```diff
@@ -0,0 +1,67 @@
+import enum
+
+class HttpResponse(enum.Enum):
+  
+  OK = (200, "success")
+  BAD_REQUEST = (400, "<h1>URL Required<h1>")
+  NOT_FOUND = (404, """
+        <html>
+            <head>
+                <title>404 Error</title>
+                <style>
+                    body {
+                        display: flex;
+                        flex-direction: column;
```

Comment:
why do we do css for this response? can we just do normal unstyled html instead? its simpler

Reply:
done

Reply:
not done

Reply:
done fr